boards/arm/mimxrt595_evk/board.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021,  NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#ifndef _BOARD_H_
+#define _BOARD_H_
+
+#include "clock_config.h"
+#include "fsl_common.h"
+
+void BOARD_FlexspiClockSafeConfig(void);
+void BOARD_SetFlexspiClock(FLEXSPI_Type *base, uint32_t src, uint32_t divider);
+void BOARD_DeinitFlash(FLEXSPI_Type *base);
+void BOARD_InitFlash(FLEXSPI_Type *base);
```

Comment:
are these functions public? why don't they follow `lower_case()` naming? I don't think we should be accepting this sort of code at board level, except this is a temporary solution until proper drivers/DT support is in place.

Reply:
Thank you for catching this, I have update the function names. This is how I see it implemented in our SDK, I have moved these functions to the `soc` folder with different function names.